tests/site/urls.py

```diff
@@ -1,11 +1,11 @@
-from django.conf.urls import include, url
-from django.contrib import admin
-
-urlpatterns = [
-    url(r"^accounts/", include("django.contrib.auth.urls")),
-    url(r"^admin/", admin.site.urls),
-    url(
-        r"^microsoft/",
-        include("microsoft_auth.urls", namespace="microsoft_auth"),
-    ),
-]
+from django.conf.urls import include, re_path
+from django.contrib import admin
+
+urlpatterns = [
+    re_path(r"^accounts/", include("django.contrib.auth.urls")),
```

Comment:
Can these to use `path` instead? Just get rid of the regex completely. It is not really needed.

Reply:
SureQ

Reply:
@AngellusMortis Can't really get rid of the regex, django 1.11 only had a regex url utility function https://docs.djangoproject.com/en/1.11/ref/urls/#

Reply:
You can remove `1.11` from the `.travis.yml` and the `tox.ini`. They are not really needed anymore since Django 1.11 is out of support.